pkg/apiserver/api_installer.go

```diff
@@ -108,6 +108,7 @@ func (a *APIInstaller) NewWebService() *restful.WebService {
 	mediaTypes, streamMediaTypes := mediaTypesForSerializer(a.group.Serializer)
 	ws.Produces(append(mediaTypes, streamMediaTypes...)...)
 	ws.ApiVersion(a.group.GroupVersion.String())
+	ws.SetDynamicRoutes(true)
```

Comment:
@lavalamp I will try to update this, but what I am really confused is whether I should go on this PR against current apiserver or I should start working on a add-on server?